libraries/MySensors/MySensor.cpp

```diff
@@ -102,7 +102,7 @@ void MySensor::begin(void (*_msgCallback)(const MyMessage &), uint8_t _nodeId, b
 
 	// Setup radio
 	if (!radio.init()) {
-		debug(PSTR("radio initialization failed!"));
+		debug(PSTR("radio initialization failed!\n"));
```

Comment:
Well, that is basically what happens then in the process() function. But yes, this approach would be also functional. I can rework it that way if you want... ;)